### PR TITLE
Add additional bridge prom metrics

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: core/services/pipeline/task.bridge.go [hunk 1]
```diff
@@ -9,12 +9,46 @@ import (
...
+	)
+	promBridgeErrors = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "bridge_errors_total",
+		Help: "Bridge error count scoped by name",
+	},
+		[]string{"name"},
+	)
+	promBridgeCacheHits = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "bridge_cache_hits_total",
+		Help: "Bridge error count scoped by name",
```
Comment (essamhassan) [inline core/services/pipeline/task.bridge.go:40]: help typo prob

File: core/services/pipeline/task.bridge.go [hunk 2]
```diff
@@ -9,12 +9,46 @@ import (
...
+	},
+		[]string{"name"},
+	)
+	promBridgeCacheErrors = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "bridge_cache_errors_total",
+		Help: "Bridge error count scoped by name",
```
Comment (essamhassan) [inline core/services/pipeline/task.bridge.go:46]: ditto

File: core/services/pipeline/task.bridge.go (hunk 1 above)
Comment (essamhassan) [reply core/services/pipeline/task.bridge.go:40]: the help field is not matching the prom metric

File: core/services/pipeline/task.bridge.go [hunk 3]
```diff
@@ -125,24 +159,29 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
...
 			return Result{Error: err}, RunInfo{IsRetryable: isRetryableHTTPError(statusCode, err)}
 		}
+		promBridgeCacheHits.WithLabelValues(t.Name).Inc()
 		lggr.Debugw("Bridge task: request failed, falling back to cache",
 			"response", string(responseBytes),
 			"url", url.String(),
 		)
 		cachedResponse = true
+	} else {
+		promBridgeLatency.WithLabelValues(t.Name).Set(elapsed.Seconds())
```
Comment (jmank88) [inline core/services/pipeline/task.bridge.go:184]: Would it be worthwhile to keep some higher precision around here? maybe millis?

Comment (samsondav) [reply core/services/pipeline/task.bridge.go:184]: This already gives high precision - duration.Seconds() returns float64 https://pkg.go.dev/time#Duration.Seconds

**Review by essamhassan — APPROVED** [review]

File: core/services/pipeline/task.bridge.go (hunk 3 above)
Comment (samsondav) [reply core/services/pipeline/task.bridge.go:184]: Note that prometheus recommends using standard units - https://prometheus.io/docs/practices/naming/

**Review by jmank88 — APPROVED** [review]

**Pull request merged**